recipes/callerpp/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -x -e -o pipefail
+
+make \
+  CC="${CC}" \
+  CFLAGS="${CPPFLAGS} ${CFLAGS} -g -Wall -O3 ${LDFLAGS}"
```

Comment:
consider using `-j ${CPU_COUNT}` - by using parallel builds you may decrease the build time

Reply:
Unfortunately there's only one source file, but it's a good suggestion nonetheless